README.md

```diff
@@ -243,6 +243,37 @@ The string format expected by the framework is a string in the `route_identifier
 }
 ```
 
+## Flaggable Feature
+
+In cases you need to have a toggle control of your feature, it's also possible to handle it through the method  `isEnabled() -> Bool` method in **Feature** protocol.
```

Comment:
```suggestion
If you need to have a toggle control for your feature, either because of a feature flag or because it has a minimum iOS version requirement, it's possible to handle it through a `Feature's` `isEnabled() -> Bool` method.
```

Improvements to English grammar + mentioning reasons this can be used